README.md

```diff
@@ -65,35 +65,35 @@ TATUM_API_KEY=""
 
 ### Command List
 
--   -ea - ethereum addressm (EVM)
--   -ba - bitcoin testnet address
--   -da - dogecoin testnet address
--   -ra - ripple testnet address
--   -s - sign sample payload using NEAR account
--   -etx - send ETH
--   -btx - send BTC
--   -dtx - send DOGE (requires API KEY)
--   -rtx - send XRP
+- -ea - ethereum addressm (EVM)
```

Comment:
Please explain formatting change, thanks!

Reply:
most likely this was due to my local vscode configuration